homeassistant/components/network/const.py

```diff
@@ -30,3 +30,5 @@
         ): vol.Schema(vol.All(cv.ensure_list, [cv.string])),
     }
 )
+
+URL_TYPES: Final = ["internal", "external", "cloud"]
```

Comment:
no need to move this into the network helper. Let's keep it around the WS command.

Reply:
Is there a reason to have this as a constant list instead of a string enum?